modules/objdetect/src/cascadedetect.cpp

```diff
@@ -1666,7 +1666,7 @@ void clipObjects(Size sz, std::vector<Rect>& objects,
     for( i = 0; i < n; i++ )
     {
         Rect r = win0 & objects[i];
-        if( r.area() > 0 )
+        if( r.empty() > 0 )
```

Comment:
Condition should be `!r.empty()` (we need non-empty rectangles)

Reply:
I will fix it this afternoon... Simpky a bad copy paste 😀